mobile_app_connector/controllers/mobile_app_controller.py

```diff
@@ -69,6 +69,21 @@ def mobile_app_contact_us(self, **parameters):
         """
         return request.env['crm.claim'].mobile_contact_us(request.jsonrequest)
 
+    @http.route(['/mobile-app-api/frequently.asked.questions/get_faq',
+                 '/mobile-app-api/frequently.asked.questions/get_faq/'],
+                type='json', auth='public', methods=['GET'])
+    def mobile_app_get_faq(self, **parameters):
+        """
+        FAQ entry point for not logged in users
+        :param parameters: all other optional parameters sent by the request
+        :return: json data for mobile app
+        """
+        odoo_obj = request.env.get('frequently.asked.questions').with_context(
```

Comment:
No need to convert language
```suggestion
        faq_handler = request.env.get('frequently.asked.questions')
```